src/report/annotation.go

```diff
@@ -137,6 +150,19 @@ func rankSeverity(f types.FindingSeverity) int {
 	return 100
 }
 
+func compareCVSSScore(a, b finding.CVSSScore) int {
+	switch {
+	case a.Score == nil && b.Score == nil:
+		return 0
```

Comment:
nit: Would this work?

```suggestion
	case a.Score == b.Score:
		return 0
```

Reply:
I think it would. Originally I had the predicates in a different order, where it didn't, and kept this version after reordering.

Reply:
Applied